### DISCUSSION TITLE
我可以在路由中使用eval吗？

### DISCUSSION THREAD

**Discussion by HeliumOctahelide** [DIYgod/RSSHub#17135] [Q&A] [answered]
我开发了一个NUXT.JS站点的路由(magic.wizards.com)，它的所有数据都放在__NUXT__变量中，是一个压缩混淆过的IIFE。由于用文本的方式解析这个IIFE太过费劲，我选择直接用eval来解析这个IIFE。这种做法是可行或推荐的吗？会不会产生安全风险？

**Comment by TonyRL** [comment, marked as answer]
You can use jsdom

**Reply by HeliumOctahelide** [reply]
I got it. Thanks!
